test/test_site/index.md

```diff
@@ -38,4 +38,33 @@ siteNav: site-nav.md
 # Include from another Markbind site
 <include src="sub_site/index.md" />
 
+# Panel without src
+<panel header="## Panel without src" expanded>
```

Comment:
The heading inside the panel is the same as the heading outside the panel. Maybe you can omit the heading outside the panel if it is the same as the one inside the panel?

This may be applicable for the ones below too.